terraform/modules/upload-service/sqs.tf

```diff
@@ -0,0 +1,58 @@
+resource "aws_sqs_queue" "upload_queue" {
+  name                      = "pre_checksum_upload_queue"
```

Comment:
Deployment stage needs to be appended to this name. Something like 'pre-checksum-upload-queue-predev' so that they don't collide between environments. In general, everything should have the environment name appended to it if its a unique name or label

Reply:
👍